tests/test_eigen.cpp

```diff
@@ -108,7 +108,7 @@ TEST_SUBMODULE(eigen, m) {
     m.def("double_threec", [](py::EigenDRef<Eigen::Vector3f> x) { x *= 2; });
     m.def("double_adscalarc", [](py::EigenDRef<VectorXADScalar> x) { x *= 2; });
     m.def("double_threer", [](py::EigenDRef<Eigen::RowVector3f> x) { x *= 2; });
-    m.def("double_adscalarr", [](py::EigenDRef<VectorXADScalarR> x) { x *= 2; });
+    // m.def("double_adscalarr", [](py::EigenDRef<VectorXADScalarR> x) { x *= 2; });
```

Comment:
Do we mean to remove this?

Reply:
Done. Fixed up the tests to have more coverage.